.github/workflows/hw2.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         hash -r
         conda config --set channel_priority strict
         conda update --all --yes
-        conda install --yes pip python numpy scipy pytest pandas matplotlib mkl-include
+        conda install --yes pip python=3.7 numpy scipy pytest pandas matplotlib mkl-include
```

Comment:
Could you please avoid adding changes that are not related to your proposal?  I'll start review after you correct the irrelevant changes.

Reply:
Sorry I didn't notice that. I've removed irrelevant changes. Thank you.